medusa/server/api/v2/config.py

```diff
@@ -129,6 +129,9 @@ def get(self, query=''):
                     'allSeasons': bool(app.DISPLAY_ALL_SEASONS),
                     'specials': bool(app.DISPLAY_SHOW_SPECIALS)
                 }
+            },
+            'showsRoot': {
+                'id': int(app.SHOWS_ROOT) if app.SHOWS_ROOT else None
```

Comment:
I know YOU only need the id but is there a reason not to just return the whole show root object/string? It'd make more sense for the API to expose it all. Think about API(third party) first before how Medusa uses it.